team-cost-reporter/aws_cost_collector/aws_cost_collector.py

```diff
@@ -0,0 +1,101 @@
+import logging.handlers
+import argparse
+import datetime
+import boto3
+import s3
+import ecs
+import cloudcheckr_format
+import json
+
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
+
+SESSION = boto3.session.Session()
+CE = SESSION.client('ce')
+
+
+def id():
+    return "cost_collector"
+
+def log (message):
+    print id() + ": " + message
+
+# get the start and end date strings based on the number of days to report
+def get_start_end_strings(number_of_days):
+    now = datetime.datetime.now()
+    n_days_ago = now - datetime.timedelta(days=number_of_days)
+
+    current_date_str = now.strftime("%Y-%m-%d")
+    n_days_ago_str = n_days_ago.strftime("%Y-%m-%d")
+
+    # start=2016-06-01
+    # end=2016-06-30
+    start_string = n_days_ago_str
+    end_string = current_date_str
+    return (start_string,end_string)
+
+
+def load_data(service, days_to_report, cloudcheckr=False, debug=False):
+    start_date, end_date = get_start_end_strings(days_to_report)
+    group_by_tag = None
+    data_full = None
+    if service == 's3':
+        group_by_tag = 'signiant:email'
```

Comment:
Remove this.  The tags are in the config file.  There should be nothing Signiant specific in the code